Bottom: 3812ac8cf839f526dbfb528746305581eb2849fa
Top:    2d29679701e4dcfb09a4bc68db546e040efc48c7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 21:50:22 +0000

Refresh of random-pile-of-improvements

---

diff --git a/src/polygon1.c b/src/polygon1.c
index f8c8832..1a12c04 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1119,13 +1119,14 @@ rounded_contour_bounds_touch (const BoxType * b, void *cl)
       /* check this vertex for any insertions */
       info.v = av;
 
-      box.X2 = (box.X1 = av->point[0] - 1) + 3; /* NB: We expand the search box to ensure we catch edges which may round to this coordinate */
+      /* NB: We expand the search box to ensure we catch edges which may round to this coordinate */
+      box.X2 = (box.X1 = av->point[0] - 1) + 3;
       box.Y2 = (box.Y1 = av->point[1] - 1) + 3;
 
       /* NB: If this actually hits anything, we are teleported back to the beginning */
       if (rtree_over->tree)
-	if (UNLIKELY (r_search (rtree_over->tree, &box, NULL, vertex_in_seg_rounded, &info)))
-	  assert (0); /* XXX: Memory allocation failure */
+        if (UNLIKELY (r_search (rtree_over->tree, &box, NULL, vertex_in_seg_rounded, &info)))
+          assert (0); /* XXX: Memory allocation failure */
     }
   while ((av = NEXT_VERTEX (av)) != &looping_over->head);
